packages/core/src/utilities/getViewportModality.ts

```diff
@@ -1,13 +1,12 @@
-import { IViewport } from '../types';
-import { StackViewport, VolumeViewport } from '../RenderingEngine';
+import type { IViewport, IStackViewport, IVolumeViewport } from '../types';
 import cache from '../cache';
 
 function getViewportModality(viewport: IViewport, volumeId?: string): string {
-  if (viewport instanceof StackViewport) {
-    return viewport.modality;
+  if ((viewport as IStackViewport).modality) {
```

Comment:
Later we might add .modality to the volume and this will break, can we properly fix this with some method? check the type?

Reply:
Actually it will be fine if we add .modality to the volume as it will then return direct.  I prefer this implementation since it correctly handles other viewport types like WSI and Video, which all have .modality.